src/select.ts

```diff
@@ -222,6 +223,19 @@ export class PrismaSelect {
         } else {
           const field = this.field(key, model)
           if (field) {
+            /**
+             * @TODO-DP: add to select all field-level required fields
+             */
+            const selectForRequiredFields = getRequiredFields(
+              modelName,
+              field.documentation || '',
+              this.options,
+              this.context
+            )
+            if (selectForRequiredFields) {
+              PrismaSelect.mergeDeep(filteredObject, { select: selectForRequiredFields })
+            }
```

Comment:
@dimaip, how do the `selectForRequiredFields` not actually end up in the returned query result?

This looks good if those `requiredFields` aren't returned, otherwise we should try and strip those back out, right?

Reply:
They end up in Prisma query results (which is what we want), but not in GraphQL response. Because of, well, GraphQL.